src/components/settingFlyout/settingFlyout.js

```diff
@@ -23,6 +23,8 @@ const currSimulationLabel = {
   false: lang.STOPPED
 };
 
+const version = '1.0.0-preview.3';
```

Comment:
could we have this value in `config.js` ?

Reply:
Done.